### ISSUE TITLE
Component: ConfirmDialog Accessibility

### ISSUE THREAD

**Issue by WTW-Jacob-Evans** [primefaces/primeng#15108]
### Describe the bug

Using the Axe DevTools, scanning a confirmation dialog will raise an accessibility issue related to a missing aria label on the inner dialog of the pop-up.

![image](https://github.com/primefaces/primeng/assets/163871134/e6627388-559a-4408-a921-642215e0d883)
![image](https://github.com/primefaces/primeng/assets/163871134/1fb3daf8-f35e-42f4-b6ca-2ca73f62a7cd)

### Environment

Windows 10

### Angular version

17.2.3

### PrimeNG version

17.11.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

**Comment by cetincakiroglu** [comment]
Fixed in https://github.com/primefaces/primeng/pull/15075
